pyproject.toml

```diff
@@ -53,21 +54,21 @@ force_grid_wrap = 2
 
 
 [tool.pylint.master]
-fail-under = 7.5
+fail-under = 6.2
 output-format = "colorized,json:.lint.json,text:.lint.txt"
 
-[tool.pylint.format]
-max-line-length = 88
-max-module-lines = 800
-
-
 [[tool.mypy.overrides]]
 module = [
-    "exasol.toolbox.sphinx.multiversion.*",
-    "test.unit.*",
-    "test.integration.*",
-]
+    "exasol.toolbox.nox.tasks",
+    "test.conftest.*",
+    "test.*",
+    ]
 ignore_errors = true
+ignore_missing_imports = true
+
+[tool.pylint.format]
+max-line-length = 88
```

Comment:
This seems a bit short. 120 would also be OK in my opinion ;)

Reply:
Ok, will update in next push.